src/config/README.md

```diff
@@ -35,4 +35,10 @@ module.
 
 The `/server` module defines the runtime configuration of the Node application
 server. It is essentially an extension of the `/env` config, but adds a few more
-host and authentication configuration values.
\ No newline at end of file
+host and authentication configuration values.
+
+## Known config keys
```

Comment:
This heading is ambiguous to me - what is a 'config key'? Are there 'unknown config keys'?

Instead, you could change this heading to "`process.env` keys supported in client code" or something, with a brief explanation like

---

## `process.env` keys supported in client code

Although `process.env` is typically only available in server-side code, our bundling process can automatically inject certain environment values into the client bundle wherever it finds a supported `process.env.YOUR_ENV_VAR` expression.

The following environment variables are supported - clients are responsible for populating them in the build environment:

Reply:
excellent will do!